addons/mail/security/mail_security.xml

```diff
@@ -96,6 +96,29 @@
             <field name="perm_unlink" eval="False"/>
         </record>
 
+        <record id="ir_rule_discuss_channel_member_is_channel_owner_unlink" model="ir.rule">
+            <field name="name">discuss.channel.member: channel owner/admins can unlink entries of their channel</field>
+            <field name="model_id" ref="mail.model_discuss_channel_member"/>
+            <field name="groups"
+                eval="[
+                    Command.link(ref('base.group_user')),
+                    Command.link(ref('base.group_portal')),
+                    Command.link(ref('base.group_public')),
+                ]"
+            />
+            <field name="domain_force">
+                [
+                    ("channel_id.channel_type", "=", "channel"),
```

Comment:
Ownership rights should also work in group. They are actually more important for groups than for channels.